upload-perf/src/upload-perf.rs

```diff
@@ -74,7 +74,6 @@ fn main() {
                         ("commit", git_commit_hash.trim().to_string(), String)
                     );
                     */
-                    
```

Comment:
🔍  What is this stray whitespace!

Reply:
:weary: this is from the new version of `cargo fmt` -- the old version forces this whitespace (not the first time this has happened to me)

Reply:
r-